src/app/js/src/SampleApp.jsx

```diff
@@ -18,6 +18,8 @@ import TextField from 'material-ui/TextField';
 import RaisedButton from 'material-ui/RaisedButton';
 import Snackbar from 'material-ui/Snackbar';
 import { Grid, Row, Col } from 'react-flexbox-grid';
+import FlatButton from 'material-ui/FlatButton';
+import { Toolbar, ToolbarGroup, ToolbarTitle } from 'material-ui/Toolbar';
```

Comment:
Nice job importing just what you need (here and in other places). This should let us take advantage of [tree-shaking](https://webpack.js.org/guides/tree-shaking/).